oat-sa/extension-tao-delivery-rdf
# Feat/aut 3623/hide features in solar design

related to 
 - https://oat-sa.atlassian.net/browse/AUT-3623
 - https://oat-sa.atlassian.net/browse/AUT-3625

⚠️ Update extension version to the release ones before merging

###Description

This PR addresses requirement `2.a` from the ticket to remove Test-Taker Panel from Deliveries page if solar design is activated.

### How to test

To test locally the dependent FE/BE branches should be specified in `composer.json`
 - [feat/AUT-3625/content-creation-mode-switch](https://github.com/oat-sa/generis/tree/feat/AUT-3625/content-creation-mode-switch)
 - [integration/RFE-1247-M3/solar-design-feature-hiding](https://github.com/oat-sa/tao-core/tree/integration/RFE-1247-M3/solar-design-feature-hiding)
 - [integration/RFE-1247-M3/solar-design-feature-hiding](https://github.com/oat-sa/extension-tao-deliver-connect/tree/integration/RFE-1247-M3/solar-design-feature-hiding)
 - [integration/RFE-1247-M3/solar-design-feature-hiding](https://github.com/oat-sa/extension-tao-testqti/tree/integration/RFE-1247-M3/solar-design-feature-hiding)
 - [feat/AUT-3623/hide-features-in-solar-design](https://github.com/oat-sa/extension-tao-delivery-rdf/tree/feat/AUT-3623/hide-features-in-solar-design)
 - [feat/AUT-3623/hide-features-in-solar-design](https://github.com/oat-sa/extension-tao-itemqti/tree/feat/AUT-3623/hide-features-in-solar-design)

Deployed to kitchen env, so functionally can be validated there: http://test-rfe-1147-facelift.playground.kitchen.it.taocloud.org:49381/
